api/openapi-spec.yaml

```diff
@@ -0,0 +1,473 @@
+openapi: 3.0.1
+info:
+  title: Event Repository REST API
+  description: Event Repository REST API for retrieving the event information
+  contact: {}
+  version: 0.0.1
+tags:
+- name: event-resource
+  description: The Event Resource API for getting single event information
+- name: search-resource
+  description: The search api provides methods for querying up/down stream given an
+    event id.
+- name: events-resource
+  description: The Events Resource API for getting all events information
+paths:
+  /events:
+    get:
+      tags:
+      - events-resource
+      summary: To get all events information
+      operationId: getEventsUsingGET
+      parameters:
+      - name: pageNo
+        in: query
+        description: "Page to display if results span across multiple pages."
+        schema:
+          type: integer
+          format: int32
+          default: 1
+      - name: pageSize
+        in: query
+        description: "The number of events to be displayed per page."
+        schema:
+          type: integer
+          format: int32
+          default: 500
+      - name: pageStartItem
+        in: query
+        description: "Intended to skip few items at the start of result, Should\
+          \ be used only if `pageNo=1` ie. first page of results."
+        schema:
+          type: integer
+          format: int32
+          default: 1
+      - name: shallow
+        in: query
+        description: "Determines if external ER's should be used to compile\
+          \ the results of query. Use `false` to use External ER's."
+        schema:
+          type: boolean
+          default: false
```

Comment:
The default value for shallow is 'true'. Please change accordingly

Reply:
Parameter was removed in a9debeb